_includes/docs/user-guide/access-token.md

```diff
@@ -25,14 +25,14 @@ One-way SSL authentication is a standard authentication mode, where your client
 ThingsBoard Team has already provisioned a valid certificate for [ThingsBoard Cloud](https://thingsboard.cloud/signup).
 Follow the [MQTT over SSL](/docs/{{docsPrefix}}user-guide/mqtt-over-ssl/) guide to provision server certificate if you are hosting your own ThingsBoard instance.
 
-Once provisioned, you should prepare a certificate chain in pem format. This chain will be used by mqtt client to validate the server certificate.
-Save the chain to your working directory as "**tb-server-chain.pem**". 
-An example of certificate chain for *mqtt.thingsboard.cloud* is located [here](/docs/paas/user-guide/resources/mqtt-over-ssl/tb-server-chain.pem).
+Once provisioned, you should prepare a CA certificate in pem format. This certificate will be used by mqtt client to validate the server certificate.
```

Comment:
"CA certificate" -> "CA root certificate" all over the changes.. See :https://en.wikipedia.org/wiki/Root_certificate